server/src/main/java/org/apache/iotdb/db/metadata/mtree/traverser/Traverser.java

```diff
@@ -137,20 +146,20 @@ protected abstract boolean processFullMatchedMNode(IMNode node, int idx, int lev
   protected void processMultiLevelWildcard(IMNode node, int idx, int level)
       throws MetadataException {
     for (IMNode child : node.getChildren().values()) {
+      traverseContext.push(node);
       traverse(child, idx + 1, level + 1);
+      traverseContext.pop();
     }
```

Comment:
If  you want to make the context contains node from root to current node's parent, maybe put this push and pop out of the for loop is better?

Reply:
all push and pop are out of loop now